tools/package-source.js

```diff
@@ -1712,6 +1711,37 @@ _.extend(PackageSource.prototype, {
     return _.keys(packages);
   },
 
+  // Returns an array of objects, representing this package's public
+  // exports. Each object has the following keys:
+  //  - name: export name (ex: "Accounts")
+  //  - arch: an array of strings representing architectures for which this
+  //    export is declared.
+  //
+  // This ignores testOnly exports.
+  getExports: function () {
+    var self = this;
+    var ret = {};
+    // Go over all of the architectures, and aggregate the exports together.
+    _.each(self.architectures, function (arch) {
+      _.each(arch.declaredExports, function (exp) {
+        // Skip testOnly exports -- the flag is intended for use in testing
```

Comment:
That's a good idea, but seems out of scope for this CL (right?). I don't think that it seems worth doing this as part of this work/release, since it causes API changes for users, that we are trying to, mostly, avoid. 

On the other hand, sending extra information to the server and back that we don't use anywhere, don't intend to, and nobody cares about, seems excessive, especially if we are planning to deprecate the entire feature. 

Here is a strawman: don't send testOnly to the server (it just seems wrong); change testOnly to only affect local packages. That doesn't affect most users, makes sense (we don't publish test packages as packages anyway, and that's the most common use case), and brings us closer to our goal (deprecate it). Though maybe it doesn't actually make sense, IDK. 

Going to think about this for a bit, but posting some thoughts in case you have some in turn.